assets/js/components/Portal.js

```diff
@@ -43,4 +53,14 @@ function Modal( { children } ) {
 	);
 }
 
-export default Modal;
+Portal.propTypes = {
+	slug: PropTypes.string,
+	appendPluginRoot: PropTypes.bool,
+};
```

Comment:
We also need to add `children` to the propTypes list.

Reply:
Added @eugene-manuilov